rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -73,6 +73,7 @@
 
     private TxHandler txHandler;
 
+    @Autowired
```

Comment:
same goes here, why would use @Autowired with @Component on same class?

Reply:
Same situation as in the previous comment, before this changes all was created and initialized in the `co.rsk.core.RskFactory`, now we would just need to create a new instance passing all the dependencies in the constructor and just return it, that's exactly what *@Component* and *@Autowired* do